### PR TITLE
Notification System Changes

### PR DESCRIPTION
**Pull request by aayush848484**
# Description and Motivation 
Updated the basic, standard and local deployment to include the following resources:

1. Logic App configured to send email for alert notification system.
2. EventHub basic tier.
...

# Change type 

- [ ] Bug fix
- [x] New feature
- [x] Enhancement
- [ ] Breaking change (breaks backward compatibility)

**Checklist:**

- [ ] All tests passed
- [x] The code follows the code style and conventions of this project
- [x] The change requires a change to the documentation
- [ ] I have updated the documentation accordingly

---
This change is [Reviewable](https://reviewable.io/reviews/azure/pcs-cli/321)

### CODE REVIEW

File: solutions/remotemonitoring/armtemplates/basic-static-map.json [hunk 1]
```diff
@@ -261,7 +275,7 @@
...
... AI.__actions as [rule.actions],\n        AI.__deviceId as [device.id],\n        DATEDIFF(millisecond, '1970-01-01T00:00:00Z', AI.__receivedTime) as [device.msg.received]\n    FROM\n        ApplyInstantRuleFilters AI PARTITION BY __partitionid\n)\n\nSELECT\n    CA.*\nINTO\n    Alarms\nFROM\n    CombineAlarms CA PARTITION BY __partitionid\n\nSELECT\n    CN.*\nINTO\n    NotificationSystem\nFROM\n    CombineAlarmNotification CN PARTITION BY __partitionid\n\nSELECT\n        CONCAT(T.IoTHub.ConnectionDeviceId, ';', CAST(DATEDIFF(millisecond, '1970-01-01T00:00:00Z', T.EventEnqueuedUtcTime) AS nvarchar(max))) as id,\n        1 as [doc.schemaVersion],\n        'd2cmessage' as [doc.schema],\n        T.IoTHub.ConnectionDeviceId as [device.id],\n        'device-sensors;v1' as [device.msg.schema],\n        'StreamingJobs' as [data.schema],\n        DATEDIFF(millisecond, '1970-01-01T00:00:00Z', System.Timestamp) as [device.msg.created],\n        DATEDIFF(millisecond, '1970-01-01T00:00:00Z', T.EventEnqueuedUtcTime) as [device.msg.received],\n        udf.removeUnusedProperties(T) as Data\nINTO\n    Messages\nFROM\n    DeviceTelemetry T PARTITION BY PartitionId TIMESTAMP BY T.EventEnqueuedUtcTime"
```
Comment (jillcary) [inline solutions/remotemonitoring/armtemplates/basic-static-map.json]: Add this query change to ASA_manager to review.

Comment (jillcary) [reply solutions/remotemonitoring/armtemplates/basic-static-map.json]: https://github.com/Azure/asa-manager-dotnet/blob/master/scripts/asa/Script.asaql

File: solutions/remotemonitoring/single-vm/setup.sh [hunk 2]
```diff
@@ -44,6 +44,9 @@ while [ "$#" -gt 0 ]; do
         --docker-tag)                   PCS_DOCKER_TAG="$2" ;;
         --evenhub-connstring)           PCS_EVENTHUB_CONNSTRING="$2" ;;
         --eventhub-name)                PCS_EVENTHUB_NAME="$2" ;;
+        --logicapp-endpoint-url)        PCS_LOGICAPP_ENDPOINT_URL="$2" ;;
```
Comment (jillcary) [inline solutions/remotemonitoring/single-vm/setup.sh]: If this is the same as the ASA event hub env vars, then we should re-use them.

File: solutions/remotemonitoring/single-vm/setup.sh [hunk 3]
```diff
@@ -208,6 +211,12 @@ echo "export PCS_AZUREMAPS_KEY=\"${PCS_AZUREMAPS_KEY}\""
 echo "export PCS_APPLICATION_SECRET=\"${PCS_APPLICATION_SECRET}\""                                       >> ${ENVVARS}
 echo "export PCS_DOCKER_TAG=\"${PCS_DOCKER_TAG}\""                                                       >> ${ENVVARS}
 echo "export PCS_LOG_LEVEL=\"${PCS_LOG_LEVEL}\""                                                         >> ${ENVVARS}
+echo "export PCS_TELEMETRY_LOGICAPP_ENDPOINT_URL=\"${PCS_LOGICAPP_ENDPOINT_URL}\""                       >> ${ENVVARS}
+echo "export PCS_TELEMETRY_EVENTHUB_CONNSTRING=\"${PCS_EVENTHUB_CONNSTRING}\""                           >> ${ENVVARS}
```
Comment (jillcary) [inline solutions/remotemonitoring/single-vm/setup.sh]: Reuse the ASA variable

**Comment by ppathan** [comment]
Stale PR, closing

**Pull request closed without merging**